### PR TITLE
feat: added delete and archive feature

### PR DESCRIPTION
**Pull request by paarthbajaj**
del archive

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -24,6 +24,24 @@
...
+}
+.archive-container {
+  display: grid;
+  margin: 3rem 3rem 0;
+  grid-template-columns: 0.6fr 1.4fr;
+  grid-template-rows: 0.3fr 1.7fr;
+  gap: 0px 0px;
+  grid-auto-flow: row;
+  grid-template-areas: "sidebar notes-list";
+}
```
Comment (munjalSaksham71) [inline src/App.css]: This two classes are almost same - Its not a good practice Please Follow DRY - Give one Common class.

File: src/context/NotesContext.js [hunk 2]
```diff
@@ -13,6 +13,47 @@ const useNotes = () => useContext(NotesContext);
...
   const encodedToken = localStorage.getItem("key");
   const [notesList, setNotesList] = useState([]);
+  const [archivedNotesList, setAchivedNotesList] = useState([]);
+  const [noteContent, setNoteContent] = useState({
+    title: "",
+    body: "",
+    in_trash: false,
+  });
+  const deleteNote = async (noteItem) => {
+    console.log(noteItem);
```
Comment (munjalSaksham71) [inline src/context/NotesContext.js:23]: Remove this Console. I Guess you don't need it now.

File: src/context/NotesContext.js [hunk 3]
```diff
@@ -13,6 +13,47 @@ const useNotes = () => useContext(NotesContext);
...
+    try {
+      const res = await axios.delete(`/api/notes/${noteItem._id}`, {
+        headers: {
+          authorization: encodedToken,
+        },
+      });
+      console.log(res);
+      // setNoteContent((note) => ({ ...note, in_trash: true }));
```
Comment (munjalSaksham71) [inline src/context/NotesContext.js:31]: Remove Comment

**Review by munjalSaksham71 — APPROVED** [review]
Looks Good, Neat Code - Gives a suggestion please check and then you are good to merge

File: src/context/NotesContext.js [hunk 4]
```diff
@@ -22,7 +63,21 @@ const NotesContextProvider = ({ children }) => {
...
+        const archivedNoteData = await axios.get("/api/archives", {
+          headers: {
+            authorization: encodedToken,
+          },
+        });
+        setAchivedNotesList(() => archivedNoteData.data);
+        console.log(archivedNoteData);
       } catch (err) {
         console.log(err);
       }
```
Comment (harshitadaswani) [inline src/context/NotesContext.js:83]: liked the use of async-await here

**Review by harshitadaswani — APPROVED** [review]
- liked how you have made the logic code simple and readable.
- liked the use of async-await here
- you can remove the non-necessary consoles from the code and then go for the merge

**Pull request merged**